token-swap/program/src/instruction.rs

```diff
@@ -92,6 +92,20 @@ pub struct WithdrawSingleTokenTypeExactAmountOut {
     pub maximum_pool_token_amount: u64,
 }
 
+/// ToggleSwap instruction data.
+#[repr(C)]
+#[derive(Debug, PartialEq)]
+pub struct ToggleSwap {
+    /// Whether to enable or disable trading.
+    pub is_trading: bool,
+
+    /// Fees for the swap. Should match the initialized fees.
+    pub fees: Fees,
+
+    /// Swap curve. Should match the initialized swap curve.
+    pub swap_curve: SwapCurve,
```

Comment:
Added b/c `Fees` and `SwapCurve` don't have the copy trait implemented--didn't dig into how to do this.

Reply:
see comment below